src/UnitTests/Traits_UnitTest.cpp

```diff
@@ -409,7 +409,6 @@ TEST_CASE("Serialization") {
     matrix(0, 0) = 1.0f;
     matrix(0, 1) = 0.0f;
     CHECK(SerializationTestImpl(Eigen::MatrixX<float>()));
-    CHECK(SerializationTestImpl(Eigen::MatrixX<float>(1, 2)));

```

Comment:
It sounds like this change is to address a build issue. If so, it should be a part of its own PR. Lets attempt to root cause the issue before making changes to remove tests.